pkgs/development/tools/purescript/spago/default.nix

```diff
@@ -16,6 +16,8 @@ let
         maintainers = (oldAttrs.maintainers or []) ++ [
           lib.maintainers.cdepillabout
         ];
+        changelog =
+          "https://github.com/purescript/spago/releases/tag/${oldAttrs.version}";
```

Comment:
We do not have a 80 characters line limit. :)

Reply:
Do we have some sort of code formatting guidelines for Nixpkgs?

Reply:
Yeah, don't add whitespace which is not necessary and decreases readability.